source/include/TypeAndDataManager.h

```diff
@@ -285,18 +285,17 @@ class TypeManager
     Int32   AQAlignment(Int32 size);
     static Int32   AlignAQOffset(Int32 offset, Int32 size) {
         if (size != 0) {
-            Int32 remainder  = offset % size;
+            const Int32 remainder  = offset % size;
```

Comment:
Adding class const modifiers to member functions is great, but I'm not sure what the point of adding const modifiers to scalar arguments or local variables which are not used in a way that is _conceptually_ constant such as this.  It would be hard to be consistent with this, and I think it adds no value.

Reply:
I agree. I'll remove uses of const for local variables.